docs/OPERATIONAL_REQUIREMENTS.md

```
@@ -0,0 +1,131 @@
+- Unconfigured integrations will revert on interaction
+- Rate limit keys act as a whitelist (see [Rate Limits](./RATE_LIMITS.md))
+
+### Withdrawal Dependencies
+
+| Withdrawal Function | Required |
+|--------------------|----------|
+| `withdrawERC4626` | Non-zero deposit rate limit for same vault |
+| `redeemERC4626` | Non-zero deposit rate limit for same vault |
+| `withdrawAave` | Non-zero deposit rate limit for same aToken |
+
+## Token Requirements
```

markdownlint-cli2 (0.20.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)